components/script/dom/performanceobserverentrylist.rs

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use dom::bindings::cell::DOMRefCell;
+use dom::bindings::codegen::Bindings::PerformanceObserverEntryListBinding;
+use dom::bindings::codegen::Bindings::PerformanceObserverEntryListBinding::PerformanceObserverEntryListMethods;
+use dom::bindings::js::Root;
+use dom::bindings::reflector::{Reflector, reflect_dom_object};
+use dom::bindings::str::DOMString;
+use dom::globalscope::GlobalScope;
+use dom::performance::PerformanceEntryList;
+use dom::performanceentry::PerformanceEntry;
+use dom_struct::dom_struct;
+
+#[dom_struct]
+pub struct PerformanceObserverEntryList {
+    reflector_: Reflector,
+    entries: DOMRefCell<PerformanceEntryList>,
+}
+
+impl PerformanceObserverEntryList {
+    fn new_inherited(entries: PerformanceEntryList) -> PerformanceObserverEntryList {
+        PerformanceObserverEntryList {
+            reflector_: Reflector::new(),
+            entries: DOMRefCell::new(entries),
+        }
+    }
+
+    #[allow(unrooted_must_root)]
+    pub fn new(global: &GlobalScope, entries: PerformanceEntryList)
+        -> Root<PerformanceObserverEntryList> {
+        let observer_entry_list = PerformanceObserverEntryList::new_inherited(entries);
+        reflect_dom_object(box observer_entry_list, global, PerformanceObserverEntryListBinding::Wrap)
+    }
+}
+
+impl PerformanceObserverEntryListMethods for PerformanceObserverEntryList {
+    // https://www.w3.org/TR/performance-timeline-2/#dom-performanceobserver
```

Comment:
Use https://w3c.github.io/performance-timeline/ for these links instead of the TR.